clang/lib/Sema/SemaExpr.cpp

```diff
@@ -2347,6 +2347,27 @@ Sema::BuildDeclRefExpr(ValueDecl *D, QualType Ty, ExprValueKind VK,
   return E;
 }
 
+// Check whether a similar function-like macro exists and suggest it
+static bool isFunctionLikeMacro(const DeclarationName &Name, Sema &SemaRef,
+                                const SourceLocation &TypoLoc) {
```

Comment:
```suggestion
static bool diagnoseFunctionLikeMacro(Sema &SemaRef, DeclarationName Name, SourceLocation TypoLoc) {
```
Needs to be re-wrapped for 80 column limit.

Reply:
I guess, clang-fmt should have done this?

Reply:
It will do it for you, I'm just used to having to point it out. :-)